src/plotApplications/helpers.js

```diff
@@ -21,7 +21,7 @@ import {
 import type {LeafletFeature, LeafletGeoJson} from '$src/types';
 import {ApplicantInfoCheckTypes, ApplicantTypes, PlotApplicationInfoCheckExternalTypes} from './enums';
 import type {RootState} from '../root/types';
-import type {PlotApplicationFormValue, ApplicationFormSection, ApplicationFormState} from './types';
+import type {PlotApplicationFormValue, ApplicationFormSection, ApplicationFormState, UploadedFileMeta} from './types';
```

Comment:
'UploadedFileMeta' is defined but never used.

Same few rows earlier (14):

'getPendingUploads' is defined but never used.

Reply:
Right, I forgot to clean up here after a bit of rearranging. Will fix.